sdk/timeseriesinsights/Azure.Iot.TimeSeriesInsights/src/Models/TimeSeries3Id.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Azure.Iot.TimeSeriesInsights
+{
+    /// <summary>
+    /// A single Time Series Id value that is an array of 3 string values that uniquely identifies a Time Series instance.
```

Comment:
can reword to "... that is part of a composite key.."

Also would be good to add an example of a composite key in json or simple terms such as Building, Floor, Room

Reply:
I like the example idea. Let's discuss offline on what that should look like.